tools/readme/readme.md.tmpl

```diff
@@ -9,6 +9,9 @@ test and build your packages. Learn about each of these and other features in [_
 
 Currently, `elastic-package` only supports packages of type [Elastic Integrations](https://github.com/elastic/integrations).
 
+Please review the [Contributing Guide](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md) to learn how to build and develop packages, understand the release procedure and
```

Comment:
Nit. Maybe we should be more specific about this contributing guide referring to packages/integrations and not about contributing to `elastic-package` itself.
```suggestion
Please review the [integrations contributing cuide](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md) to learn how to build and develop packages, understand the release procedure and
```